docs/packages/add-new-package-to-package-repo.md

```diff
@@ -6,19 +6,20 @@ requirements of a package being in a particular package repository.
 
 ## Steps to add a package to a package repository
 
-Let's take the example of adding a package to `management` package repository, below steps illustrate how that can be done
+Let's take the example of adding a package to `management` package repository.
+Below are steps to illustrate how that can be done.
 
 1. Scaffold a new package and update the package files
 
    ```shell
       PACKAGE_REPOSITORY=management PACKAGE_NAME=my-package make create-package
```

Comment:
we can remove `PACKAGE_REPOSITORY` var here

Reply:
Done